pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: browser_launcher
+description: A standardized way to launch web browsers.
+
+version: 0.1.0-dev
+
+author: Dart Team <misc@dartlang.org>
+homepage: https://github.com/dart-lang/browser_launcher
+
+environment:
+  sdk: '>=2.2.0-dev <3.0.0'
```

Comment:
Drop the `-dev` - https://travis-ci.org/dart-lang/browser_launcher/jobs/524237824

Reply:
done